### PR TITLE
Skyblock -> SkyBlock (except SBA)

### PR DESCRIPTION
**Pull request by CalMWolfs**
this is all search showed me but there may be more idk

### CODE REVIEW

File: FEATURES.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ Use `/sh` or `/skyhanni` to open the SkyHanni config in game.
 + Saves missing items from canceled buy orders to clipboard for faster re-entry.
 + Update Timer showing when the next api data update happens.
 + Added Price Website button. - hannibal2
-    + Adds a button to the bazaar product inventory that will open the item page in skyblock.bz.
+    + Adds a button to the bazaar product inventory that will open the item page in SkyBlock.bz.
```
Comment (hannibal002) [inline FEATURES.md]: This is an URL, no need to add lowercase chars here.

File: src/main/java/at/hannibal2/skyhanni/features/misc/discordrpc/DiscordRPCManager.kt [hunk 2]
```diff
@@ -170,7 +170,7 @@ object DiscordRPCManager : IPCListener {
             },
             5,
             TimeUnit.SECONDS
-        ) // wait 5 seconds to check if the new world is skyblock or not before stopping the function
+        ) // wait 5 seconds to check if the new world is SkyBlock or not before stopping the function
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/features/misc/discordrpc/DiscordRPCManager.kt]: The word is literally "skyblock"

**Review by hannibal002 — CHANGES_REQUESTED** [review]
I did most of your changes myself already, didn't notice your pr sadly

File: FEATURES.md (hunk 1 above)
Comment (CalMWolfs) [reply FEATURES.md]: thought i avoided all urls i guess this one slipped through

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**